src/edges_estimate/foregrounds.py

```diff
@@ -220,6 +220,22 @@ def model(self, **p):
         return p["amp"] * np.sin(2 * np.pi * self.freqs / p["lambda"] + p["phase"])
 
 
+@attr.s
+class Sinusoid_cossin(Foreground):
+    base_parameters = [
+        Parameter("amp0", 0, min=0, max=1, latex=r"A_{\rm sin}"),
```

Comment:
maybe call the parameter `amp_sin`? Also, the `min` and `max` are meant to be absolute limits on the parameter (not for any particular problem, but in general). Are you sure that the amplitude should have a max of one?

Reply:
For that matter, are you sure you shouldn't allow negative amplitudes?